app/services/github/repositories/sync.rb

```diff
@@ -32,6 +27,33 @@ def call
       def repositories
         @repositories ||= Repository.all
       end
+
+      def repository_owner_and_name(repository)
+        repository.link.split('/')[-2..-1]
+      end
+
+      def update_repository_languages(repository)
+        repository_owner, repository_name = repository_owner_and_name(repository)
+        request = client.repos.languages(repository_owner, repository_name)
+
+        if request.success?
+          languages = request.body.keys[0..MAX_LANGUAGES_NUMBER-1].join(', ')
+
+          repository.update(language: languages)
```

Comment:
Maybe it's a good idea to use the `bang!` method here.
```suggestion
          repository.update!(language: languages)
```

https://github.com/rubocop/rails-style-guide#save-bang

Reply:
Thanks for the tip!